frontend/server/src/DAO/Assignments.php

```diff
@@ -56,6 +56,9 @@ public static function getAssignmentsProblemsStatistics(
             bpr.problem_alias,
             VARIANCE(bpr.max_user_score_for_problem) AS variance,
             AVG(bpr.max_user_score_for_problem) AS average,
+            AVG(
+                CASE WHEN bpr.max_user_percent_for_problem = 1 THEN 1 ELSE 0 END
```

Comment:
bajo qué condiciones esto podría ser nulo? se puede hacer algo para que no sea nulo? (quizás cambiando el `COALESCE` de L80-L82 por `IFNULL`, según yo los otros lugares donde lo usamos dejan de causar problemas con los nulos).

también, con el cambio que @carlosabcs está haciendo para dar puntos extras, esto quizás sea bueno cambiar por

```suggestion
                CASE WHEN bpr.max_user_percent_for_problem >= 1 THEN 1 ELSE 0 END
```